packages/@webex/internal-plugin-device/src/device.js

```diff
@@ -363,15 +364,15 @@ const Device = WebexPlugin.extend({
 
   // Registration method members
 
-  /* eslint-disable require-jsdoc */
   /**
    * Refresh the current registered device if able.
    *
+   * @param {CatalogDetails} includeDetails - The details to include in the refresh/register request.
    * @returns {Promise<void, Error>}
    */
   @oneFlight
   @waitForValue('@')
-  refresh() {
+  refresh(includeDetails = CatalogDetails.all) {
```

Comment:
defaults to 'all' for backwards compatibility

Reply:
would be nicer to be as an object {} in case we add anything in future , would need less changes on web client